mypy.ini

```diff
@@ -8,6 +8,3 @@ disallow_untyped_decorators = False
 ; This is needed because version 1.20.0 of NumPy introduces typing, so some ignores are necessary /
 ; unnecessary depending on the version of numpy:
 warn_unused_ignores = False
-; It would actually be good to disallow implicit reexport, but that will take some effort to be
-; strict about our use of __all__.
-implicit_reexport = True
```

Comment:
Don't you need to explicitly set this to False? (the docs suggest it defaults to True)

Reply:
> Don't you need to explicitly set this to False? (the docs suggest it defaults to True)

The `strict = True` above sets this to `False`.